cypress/integration/plugins/index-management-dashboards-plugin/rollups_spec.js

```diff
@@ -17,7 +17,7 @@ describe('Rollups', () => {
     cy.visit(`${BASE_PATH}/app/home#/tutorial_directory/sampleData`);
 
     // Click on "Sample data" tab
-    cy.contains('Sample data').click({ force: true });
+    cy.contains('Sample data', { timeout: 60000 }).click({ force: true });
```

Comment:
You don't need this line.
There's not en element contains text "Sample data" on that page `${BASE_PATH}/app/home#/tutorial_directory/sampleData`. No matter how long you wait, it will fail because it won't find any element. 
![image](https://github.com/opensearch-project/opensearch-dashboards-functional-test/assets/32652829/b0bf7b5a-f053-444f-9a88-ca7b441f7483)

Reply:
@zhongnansu Am I missing something here? Isn't `Sample data` there(highlighted)?
<img width="921" alt="image" src="https://github.com/opensearch-project/opensearch-dashboards-functional-test/assets/20185657/819999bf-f9f5-4da2-8c25-850f4700d8b0">